src/server/plugins/engine/pageControllers/PageControllerBase.ts

```diff
@@ -472,6 +476,18 @@ export class PageControllerBase {
 
       viewModel.backLink = this.getBackLink(progress)
 
+      if (progress[0]) {
```

Comment:
This is not correct. How can I correctly identify first page to satisfy below

`Should appear on the first question page when previewing a form and the check your answers page if email address is missing.` 

https://dev.azure.com/defragovuk/DEFRA-CDP/_workitems/edit/419250#19509961

Reply:
I was testing wrong scenario. This issue is resolved.